src/telegramify_markdown/render.py

```diff
@@ -238,7 +238,9 @@ def render_list_item(
     ) -> Iterable[str]:
         token_origin = str(token.leader).strip()
         if token_origin.endswith("."):
-            token.leader = formatting.escape_markdown(token.leader) + " "
+            if not token.leader.endswith(" "):
```

Comment:
**I found that since tokens cannot be deeply copied, the render function will modify the token every time it determines whether the limit is exceeded.
The longer the text, the more modifications are made.**